app/build.gradle

```diff
@@ -60,7 +60,7 @@ android {
 //            PRIVATE BUILD: comment (add two slashes at the start of each line) all the `sentry`
 //            block for private release builds support
             sentry { // https://docs.sentry.io/platforms/android/#gradle-configuration
-                autoProguardConfig true
```

Comment:
I don't think that disabling proguard is a smart idea.

Reply:
It's not disabling proguard, it's just sentry settings. It was even removed at 2.0.0 version because wasn't used. https://github.com/getsentry/sentry-android-gradle-plugin/issues/30
Also updated our sentry gradle plugin to the last version